website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md

```diff
@@ -12,4 +12,13 @@ Users of the Discovery API can now query the latest state of their environment,
 - [About the Discovery API](/docs/dbt-cloud-apis/discovery-api)
 - [Use cases and examples for the Discovery API](/docs/dbt-cloud-apis/discovery-use-cases-and-examples)
 - [Query the Discovery API](/docs/dbt-cloud-apis/discovery-querying)
-- [Project state in dbt Cloud](/docs/deploy/project-state#project-state-in-dbt-cloud) 
\ No newline at end of file
+- [Project state in dbt Cloud](/docs/deploy/project-state#project-state-in-dbt-cloud) 
+
+## Roadmap
+
+dbt Labs is continually enhancing the dbt Cloud Discovery API to ensure you have access to timely, rich, and reliable metadata about your dbt runs. In the coming year, we’ll expand the range of questions the API helps answer so you can more easily understand the state, meaning, and structure of your data to inform data development and analysis experiences.
+
+- **[Now] Query across jobs & lineage** &mdash; Get the latest state of a dbt DAG (production environment) to find, understand, and trust the right dataset to analyze.
+    - **[Ongoing] Improvements** &mdash; Enhanced developer ergonomics, state fidelity, and metadata timeliness.
```

Comment:
Did you mean for this to be indented? I'm just not sure if these things will be updated on an ongoing basis or they were updated recently and there will be other ongoing improvements.